examples/IRrecvDumpV2/IRrecvDumpV2.ino

```diff
@@ -62,6 +62,11 @@ void  encoding (decode_results *results)
 //
 void  dumpInfo (decode_results *results)
 {
+  if (results->overflow) {
+    Serial.println("IR code too long. Edit IRremoteInt.h and increase RAWBUF");
```

Comment:
This is currently how it is implemented in the Arduino library. 
To be able to tell number of elements we need another counter in the interrupt handler to keep counting interrupts. new counter because rawlen should always be less than rawbuf size as it used for telling number of elements in rawbuf and we shouldn't change this semantics.
Displaying part of the data has no use for general purpose but only for debugging.
Maybe we should think about how to add it as part of if DEBUG is set and of course should be in a new commit anyway.